### PR TITLE
added new data-erasure route

### PR DESCRIPTION
**Pull request by cigar-galaxy82**
https://user-images.githubusercontent.com/56212958/108580097-04cc2e00-7350-11eb-905d-d7ce6890e866.mp4

### CODE REVIEW

**Comment by bkimminich** [comment]
Awesome! I'll test the new screen and review the code next week! First impression is very good! 👍

File: test/api/dataErasureApiSpec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const URL = 'http://localhost:3000'
+
+describe('/datarasure', () => {
+  it('GET data-erasure page', () => {
+    return frisby.get(URL + '/dataerasure')
+      .expect('status', 200)
+  })
+
+  it('GET data-erasure page', () => {
```
Comment (bkimminich) [inline test/api/dataErasureApiSpec.js:16]: Copy/paste mistake?

File: frontend/src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -131,7 +131,7 @@
         </mat-icon>
         <span translate>TITLE_REQUEST_DATA_EXPORT</span>
       </button>
-      <button mat-menu-item [routerLink]="['privacy-security/erasure-request']" aria-label="Go to data subject page">
+      <button mat-menu-item (click) = "goToDataErasurePage()" aria-label="Go to data subject page">
```
Comment (bkimminich) [inline frontend/src/app/navbar/navbar.component.html:134]: Similar change is also needed in sidenav component (used on smaller screens)

File: views/data-erasure.hbs [hunk 3]
```diff
@@ -0,0 +1,183 @@
+<meta charset="UTF-8">
```
Comment (bkimminich) [inline views/data-erasure.hbs:1]: Clicking on "Delete user data" button without entering any data gives a funny screen duplication effect. Also works multiple times in a row, see screenshot:
image

File: views/data-erasure.hbs [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        <h5>Request Data Erasure</h5>
+        <form onsubmit='return onSubmit(this)'>
+        <div class="mt-2">
+		<input type="email" placeholder="email" name="email" class="w-full border border-black rounded outline-none p-2">
+		</div>
+		<div class="mt-2">
+		<input type="text" name="answer" placeholder="Security Answer" class="w-full border border-black rounded outline-none p-2">
+		</div>
+		<div class="mt-4 flex justify-end">
+		<button class="uppercase px-8 py-2 rounded border border-green-600 text-green-600 max-w-max shadow-sm hover:shadow-lg" type="submit">Delete User Data</button>
```
Comment (bkimminich) [inline views/data-erasure.hbs]: On the Angular screen there was a feedback message (in the snackbar) displayed when the deletion request was sent. There is no more feedback for the user from the HBS screen, not when returning to the Angular client also not on the HBS screen itself.

**Review by bkimminich — CHANGES_REQUESTED** [review]

File: views/data-erasure.hbs (hunk 4 above)
Comment (cigar-galaxy82) [reply views/data-erasure.hbs]: I display the message "Sorry to see to leave" with the go to home button in place of that

File: test/api/dataErasureApiSpec.js (hunk 1 above)
Comment (cigar-galaxy82) [reply test/api/dataErasureApiSpec.js:16]: yes I will change the code then create a new pull request

File: frontend/src/app/navbar/navbar.component.html (hunk 2 above)
Comment (cigar-galaxy82) [reply frontend/src/app/navbar/navbar.component.html:134]: solved

File: views/data-erasure.hbs (hunk 3 above)
Comment (cigar-galaxy82) [reply views/data-erasure.hbs:1]: I added a feature where button will be disabled until both the input block contain value

**Pull request closed without merging**